go.mod

```diff
@@ -24,8 +24,11 @@ require (
 
 replace k8s.io/client-go => k8s.io/client-go v0.20.5
 
-// below: temporary measure to include multicloud-operators-foundation project with ManagedClusterView through forked repo
-require github.com/tjanssen3/multicloud-operators-foundation/v2 v2.0.0-20210512222428-660109db7f1d
+require (
+	github.com/prometheus/common v0.10.0
```

Comment:
(note to self) Address this in the above require block in  https://github.com/RamenDR/ramen/pull/189